### PR TITLE
5.5.0 release preview

### PR DESCRIPTION
**Pull request by Cryptkeeper**
**5.5.0** *(May 11 2020)*

**IMPORTANT**
This update moves ping timestamps to a shared timestamp per round. Meaning that when pinging servers, each will share the same timestamp for that series of pings. The legacy backend used a timestamp per ping per series of pings. This means after updating Minetrack, the historical graph may render slightly inaccurate for the first 24 hours (or whatever your config.json->graphDuration is), and will automatically correct itself as it receives new updates. Don't worry.

- Replaces flot.js charts with uPlot charts. This new chart library renders much quicker and supports a reduced data format. This results in ~1/12th the bandwidth use when sending the historical graph. 
- Removed jQuery (flot.js required this dependency). Between removing flot.js and jQuery, the page size has been reduced by 100KB (33%)!
- New historical graph tooltip design to better compare multiple servers.
- Historical graph now supports click dragging to zoom in to a custom time frame. Double click to reset.
- Historical graph now displays time markers along the bottom.
- All graphs now have horizontal ticks to improve readability.
- Graphs will now display gaps (null) when the ping fails. This removes legacy graph smoothing code and prevents 0 player count pings messing up graph scales.
- Graphs will now render the same on initial page load as they will after being open for a while. This fixes a long standing bug where the frontend ignored 0 player count pings in updates but not on initial load.
- Removes the mobile browser detection/manual historical graph load request. It is now automatically loaded given its smaller size.

Faster, smaller, more features.

### CODE REVIEW

File: lib/database.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const sqlite = require('sqlite3')
```
Comment (leeoniya) [inline lib/database.js:1]: @Cryptkeeper https://github.com/JoshuaWise/better-sqlite3 ;)

Comment (Cryptkeeper) [reply lib/database.js:1]: Yes it does @leeoniya :) I'm thrilled to get rid of flot.js + its jQuery dependency for something speedier.

Food for thought, a proper distribution method for uPlot (cdnjs?) would be great. I feel quite hackish by copying the files and serving them myself. 

Thanks for the shout on better-sqlite3, I'll check it out!

Comment (leeoniya) [reply lib/database.js:1]: > Food for thought, a proper distribution method for uPlot (cdnjs?) would be great. I feel quite hackish by copying the files and serving them myself.

https://www.npmjs.com/package/uplot
https://unpkg.com/uplot/dist/uPlot.iife.min.js
https://unpkg.com/uplot/dist/uPlot.min.css

personally, i'd recommend the npm route, using the ESM build & bundling with Rollup & Terser.

Comment (Cryptkeeper) [reply lib/database.js:1]: I completely overlooked that, thanks again.

**Pull request merged**